runelite-client/src/main/java/net/runelite/client/ui/components/colorpicker/CheckerboardBackground.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020, Martin H <pilino@posteo.de>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.ui.components.colorpicker;
+
+import java.awt.Color;
+import java.awt.Graphics;
+
+public class CheckerboardBackground
+{
+	private static final int CHECKER_SIZE = 10;
+
+	public static void paintCheckerboard(Graphics g, int width, int height)
```

Comment:
This seems like it is in an odd place since it is also used outside of the colorpicker but I'm not sure where a better place would be? Maybe just duplicate it both places?

Reply:
@Nightfirecat asked me to unify the duplicate code into a single public class: https://github.com/runelite/runelite/pull/12354#discussion_r477614502

Reply:
I'd probably rather have it duplicated since it is so small than have the backwards dep of ui components -> colorpicker